support-files/rpm/server-postin.sh

```diff
@@ -1,3 +1,7 @@
+%if %{with init_systemd}
+%systemd_post %{daemon_name}.service
```

Comment:
Are these systemd macros guaranteed to exist even on the oldest rpm-based systemd distro that we support?